src/Test/Verilog/SVType.idr

```diff
@@ -293,12 +293,12 @@ namespace SVType
     -- Packed struct, union, enum
 
   public export
-  data State4S : IntegerVectorType -> Type where
+  data State4S : Atom -> Type where
```

Comment:
I believe the `S` suffix of the `State4S` type name initially came from the `Scalar` naming of Atoms. Should we change it to `State4A`?

Reply:
This is reasonable. Some other related types could be renamed same way